app/models/name/parse.rb

```diff
@@ -462,7 +462,7 @@ def format_autonym(name, author, _rank, deprecated)
         format_name(words[-1], deprecated)
       ].join(" ")
     else
-      format_name(name, deprecated) + " " + author
+      "#{format_name(name, deprecated)} #{author}"
```

Comment:
@mo-nathan will any of the changes to this file mess up things yo're working on?

Reply:
Don't think so.  Happy to resolve any conflicts if they show up.